spec/support/shared_examples/abilities/agency_with_refund_examples.rb

```diff
@@ -4,4 +4,8 @@
   it "should be able to view revoked reasons" do
     should be_able_to(:view_revoked_reasons, WasteCarriersEngine::RenewingRegistration)
   end
+
+  it "should be able to refund a payment" do
```

Comment:
Can we also test that the finance users and the "below agency_with_refund examples" cannot do this?

Reply:
Oh yes! Good spot :)